wcfsetup/install/files/lib/system/page/handler/AbstractMenuPageHandler.class.php

```diff
@@ -37,4 +44,20 @@ public function isVisible($objectID = null)
     public function cacheObject(int $objectID): void
     {
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function setMenuItem(MenuItem $menuItem): void
+    {
+        $this->menuItem = $menuItem;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getMenuItem(): MenuItem
+    {
+        return $this->menuItem;
```

Comment:
This will fail hard if `$this->menuItem` is not set yet. You _could_ work around this by changing the return type to `?MenuItem` and change the body to `return $this->menuItem ?? null;`.